bot.py

```diff
@@ -14,7 +14,7 @@
 with open("config.yaml") as file:
     config = yaml.load(file, Loader=yaml.FullLoader)
 
-intents = nextcord.Intents.default()
+intents = nextcord.Intents.default().all()

```

Comment:
This gives the bot all the intents needed. Likely not needed for the main bot but makes making testing bots easier.